src/pages/Timeline/index.js

```diff
@@ -64,11 +69,31 @@ export default class Timeline extends Component {
   }
 
   _fetchTweets = () => {
-    if (!this._tweetStream.loading) {
-      this._tweetStream.load(tweets => {
-        tweets = this.state.tweets.concat(tweets);
-        this.setState({ tweets });
-      });
+    let { isLoggedIn, tweets } = this.state;
+    let url = config.tweetsServerUrl + '/tweets/0';
```

Comment:
nit: Seems a little odd to pass _either_ a Tweet id _or_ "0" to the service. Maybe we could make the routes either `/tweets` (for the first page) or `/tweets/:id`?